wordpress-mobile/WordPress-iOS
# [Hack Week] Post success view

Fixes #https://github.com/wordpress-mobile/WordPress-iOS/issues/21509.

I've been going through the maintenance board, noticed the linked defect, and decided to resurrect this PR from the last Hack Week, which I didn't get to complete then. The new screen looks more like what you see in Gutenberg, and has a new "Promote with Blaze" button.

The initial idea was to show this automatically as part of the Prepublishing sheet, but this is a more conservative approach that simply replaced the current screen (with a slightly broken layout).

The design comes from one of the prototypes made with Chris, but with some minor changes. There've been so many versions of this, I don't any particular direction was picked as the final one in the end.

## To test:

- Publish a new post
- Tap "View" on a snackbar
- ✅ Verify that the updated success screen is shown
- ✅ Verify that all buttons are working
- ✅ Verify that for Blaze-enabled blogs, "Promote with Blaze" button is visible.


## Regression Notes
1. Potential unintended areas of impact: Post Publish Success Notice View
2. What I did to test those areas of impact (or what existing automated tests I relied on): manual
3. What automated tests I added (or what prevented me from doing so): n/a

PR submission checklist:


Testing checklist:
